src/wallet/rpcwallet.cpp

```diff
@@ -1515,6 +1522,7 @@ static UniValue listsinceblock(const JSONRPCRequest& request)
                 RPCResult{
             "{\n"
             "  \"transactions\": [\n"
+            "    \"involvesWatchonly\": xxx, (bool) Only returned if imported addresses were involved in transaction\n"
```

Comment:
"Only returned" means what? Maybe it's just me, but this and the rest of the sentence isn't clear.

nit: missing period at end of sentence.